src/main/java/seedu/address/ui/CommandBox.java

```diff
@@ -59,7 +77,27 @@ private void handleCommandEntered() {
      * Sets the command box style to use the default style.
      */
     private void setStyleToDefault() {
+        commandTextIcon.setIconColor(Color.WHITE);
         commandTextArea.getStyleClass().remove(ERROR_STYLE_CLASS);
+        commandTextArea.setStyle("-fx-text-fill: white;");
+    }
+
+    /** Sets the command box style to indicate the command box is focused. */
+    private void setStyleToFocused() {
+        commandTextIcon.setIconColor(Color.WHITE);
+        if (commandTextArea.getStyleClass().contains(ERROR_STYLE_CLASS)) {
+            return;
+        }
+        commandTextArea.setStyle("-fx-text-fill: white;");
+    }
+
+    /** Sets the command box style to indicate the command box is not focused. */
+    private void setStyleToUnfocused() {
+        commandTextIcon.setIconColor(Color.GREY);
+        if (commandTextArea.getStyleClass().contains(ERROR_STYLE_CLASS)) {
+            return;
+        }
+        commandTextArea.setStyle("-fx-text-fill: grey;");
     }
```

Comment:
Good use of abstraction and alignment to the pre-existing setStyleToDefault method!